### PR TITLE
TR-1 / CT-1

### PR DESCRIPTION
**Pull request by torinvdb**
Initial draft PR to add TR-1 / CT-1. Please let me know if the content and/or format is okay, so I can make any changes necessary.

### CODE REVIEW

**Comment by torinvdb** [comment]
Considering adding some more specific frameworks and/or tooling for this based on https://github.com/finos/ai-readiness/issues/27. What is the general guidance here?

File: docs/_threats/tr-1.md [hunk 1]
```diff
@@ -9,4 +9,16 @@ external-refs:
...
+
+LLMs can retain data from training processes or user interactions, potentially recalling sensitive information during unrelated sessions, a phenomenon known as "memorization." When data such as Personally Identifiable Information (PII) or proprietary financial strategies enter the model, the risk of inadvertent disclosure rises, particularly when insufficient data sanitization or filtering mechanisms are in place. Additionally, adversarial actors could exploit prompt injection attacks to manipulate the model into revealing sensitive data.
```
Comment (ColinEberhardt) [inline docs/_threats/tr-1.md:14]: Regarding memorization, this is an interesting point, and one where I think there is some confusion. There is a common misconception that all hosted LLMs are training on all the input data (prompts) sent to them. This is not the case, for some (e.g. ChatGPT) you can opt-out, for enterprise / commercial products they don't use your data for training.

I think we need to draw a clear distinction here between the risks associated with either free hosted LLMs, and the more enterprise products.

And of course, highlight teh risk of not reading the terms!

Comment (torinvdb) [reply docs/_threats/tr-1.md]: Just added an additional paragraph that better explains this consideration. Let me know if you have any further suggestions!

**Comment by ColinEberhardt** [comment]
thanks for addressing the review feedback :-)

**Pull request merged**